pkg/sql/conn_executor.go

```diff
@@ -2945,12 +2945,10 @@ func (ex *connExecutor) execCopyIn(
 		initPlanner: func(ctx context.Context, p *planner) {
 			ex.initPlanner(ctx, p)
 		},
-		resetPlanner: func(ctx context.Context, p *planner, txn *kv.Txn, txnTS time.Time, stmtTS time.Time) {
```

Comment:
should we delete this object from the struct too?

Reply:
oops my fix is invalid. i forgot about the mode where we batch copies and call this `resetPlanner` in between each batch.